examples/kops-kube-vpc/Makefile

```diff
@@ -0,0 +1,80 @@
+# Hardcoding value of 3 minutes when we check if the plan file is stale
+STALE_PLAN_FILE := `find "tf.out" -mmin -3 | grep -q tf.out`
+SHELL := /bin/bash
+
+AWS_AVAILABILITY_ZONE := `terraform output -json | jq '.availability_zone.value' | tr -d '"'`
+KOPS_STATE_BUCKET := `terraform output -json | jq '.state_bucket_id.value' | tr -d '"'`
+KOPS_CLUSTER_NAME := `terraform output -json | jq '.subdomain_zone_name.value' | tr -d '"'`
+KOPS_CLUSTER_ZONE := `terraform output -json | jq '.subdomain_zone_id.value' | tr -d '"'`
+KOPS_AWS_VPC := `terraform output -json | jq '.vpc_id.value' | tr -d '"'`
+
+## Check if tf.out is stale (Older than 2 minutes)
+check-plan-file:
+	@if ! ${STALE_PLAN_FILE} ; then \
+		echo "ERROR: Stale tf.out plan file (older than 3 minutes)!"; \
+		exit 1; \
+	fi
+
+## Runs terraform get and terraform init for env
+init:
```

Comment:
The Makefile should inclue the `require-SOME_ENV_VAR` target if it uses any env vars.

Reply:
This helps us to not run things accidentally with an undefined env....who knows what might happen.